roboteam_ai/src/skills/ballPlacement/BallPlacementReceive.cpp

```diff
@@ -18,25 +18,21 @@ bt::Node::Status BallPlacementReceive::onUpdate() {
         return Status::Failure;
     }
 
-    if (world::world->robotHasBall(robot->id, true)) {
+    if (coach::g_pass.isPassed() && ball->vel.length() <= Constants::BALL_STILL_VEL()) {
         return Status::Success;
     }
 
-    if (ball->pos.dist(coach::g_ballPlacement.getBallPlacementPos()) < 0.5) {
-        publishRobotCommand();
+    if ((ball->pos - coach::g_ballPlacement.getBallPlacementPos()).length() < 0.15) {
         return Status::Success;
     }
 
     if (coach::g_pass.isPassed()) {
-        // Check if the ball was deflected
-//        if (passFailed()) {
-//            publishRobotCommand(); // halt
-//            return Status::Running;
+//        if(ballDeflected()) {
+//            return Status::Failure;
 //        }
```

Comment:
no commented code please; also why is this if statement empty? This seems like a testing artifact or something else weird.